package.json

```diff
@@ -79,20 +74,29 @@
     "webpack-merge": "5.8.0"
   },
   "scripts": {
-    "build": "grunt",
-    "hooks": "\\cp .github/hooks/pre-push .git/hooks/pre-push",
-    "lint": "npm run type-check && npm run lint:js && npm run lint:tests && npm run lint:styles",
-    "lint:js": "eslint './src/js' --ext .ts --ext .js",
+    "clean": "shx rm -rf ./bin-* && shx mkdir -p ./bin-release ./bin-debug",
+    "build": "npm-run-all -s clean -p build:*",
+    "build:js": "webpack --config webpack.config.js",
+    "build:css": "npm-run-all build:css:less build:css:postcss",
+    "build:css:less": "node scripts/build-css.js",
+    "build:css:postcss": "postcss bin-debug/css/*.css --use autoprefixer --replace",
+    "build:notice": "node scripts/build-notice.js",
+    "start": "npm-run-all -p watch serve",
```

Comment:
I tried most of these scripts but I am running into issues with the `start` and `serve` commands:
<img width="901" alt="npm run serve" src="https://user-images.githubusercontent.com/27169173/151037317-a832ada3-138e-4901-8bb9-7502b13f4284.png">
<img width="894" alt="npm run start" src="https://user-images.githubusercontent.com/27169173/151037322-69044b4c-0c84-47b0-8fb7-1ffd853ee548.png">

I've tried deleting my node_modules and package-lock.json before running `npm install` but i'm still seeing this behavior on my end.

Reply:
sorry about that I ended up installing `static-server` rather than `serve-static` which is why you get this issue. I am working on a fix right now.

Reply:
@Dawolee This should be fixed now. Not entirely sure why we even had a server though there is only an index.html at `localhost:3000/test/manual`

Reply:
Okay! Can confirm both commands work now. Not sure about the functionality about the `serve`/`start` here vs in commercial though hmm..